api/Navigator.json

```diff
@@ -1374,7 +1374,8 @@
             },
             "firefox": {
               "version_added": "2",
-              "version_removed": "59"
+              "version_removed": "62",
+              "notes": "Deprecated in Firefox 59."
```

Comment:
I think it would be good to omit this note. This whole feature is marked as deprecated anyway